flake.nix

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
This looks indeed weird. I have a look later.

Reply:
Thanks, send a message here if I need to do something.